src/tests/multihost/alltests/test_kcm.py

```diff
@@ -119,3 +119,57 @@ def test_kcm_check_socket_path(self, multihost, backupsssdconf):
               "and.sssd.conf...some_path.kcm.socket..don't.match"
         find = re.compile(r'%s' % msg)
         assert find.search(log)
+
+    @pytest.mark.tier1_2
+    def test_expired_tickets(self, multihost, backupsssdconf):
+        """
+        :title: IDM-SSSD-TC: kcm_provider: Expired tickets are removed
+        :id: db532785-a00f-4be9-b413-592e0550fe9c
+        :setup:
+         1. Configure short ticket lifetime in krb5.
+         2. Create an user.
```

Comment:
"create a user" 

When "u" makes the same sound as the "y" in you, or "o" makes the same sound as "w" in won, or "e" makes the sound of "u" then "a" is used.

A United
A Union

English is weird.

Reply:
fixed